src/ralph/api/routers/statements.py

```diff
@@ -61,11 +63,34 @@
 }
 
 
+def _parse_agent_parameters(agent_obj: dict):
```

Comment:
We could even LRU-cache this method, WDYT?

Reply:
Good idea ! I will add this.